### PR TITLE
contacts: replacing contact-store/hook

### LINKED ISSUE #1305: Ability to add ships to the contact store
At the moment we only have contact-store data that was added via old groups. We are not adding ships that are newly interacted with in groups/talk to the contact store.

### PR DESCRIPTION
**Pull request by joemfb**
Fixes #1305.

- [ ] subscription sanity checks and repair
- [x] reflect `%meet` in `/news`
- [x] reflect `%drop` in `/news`
- [x] `s/resource/flag/g`
- [x] update frontend to support (new) new interface
- [ ] reflect subscription state in json conversions ("stale" flag)
- [ ] remove contacts on breach
- [ ] moar livenet testing

### CODE REVIEW

**Comment by patosullivan** [comment]
Seeing an issue where changes to a ships own profile don't seem to update for the user:

https://user-images.githubusercontent.com/1221094/229592885-954e659b-373f-46d2-820c-a0ff36be8f0b.mov

**Comment by jamesacklin** [comment]
Check the agent it's subscribing to? Since we're in this interim state, things might not be 100% hooked up.

Does %dbug state show the change?

**Review by philipcmonk — CHANGES_REQUESTED** [review]
I have not reviewed any typescript, but the hoon looks good except this one thing that needs to be fixed.

**Comment by jamesacklin** [comment]
Excellent, thank you @philipcmonk

@arthyn Are you good to make this change or do you want to tag someone in?

File: desk/app/contacts.hoon [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+                groups=(set resource)
+                last-updated=@da
+            ==
+          --
+        --
+    ::
+    ^+  cor
+    =/  bas  /(scot %p our.bowl)/contact-store/(scot %da now.bowl)
+    ?.  .^(? gu+bas)  cor
+    =/  ful  .^(rolodex:legacy gx+(weld bas /all/noun))
```
[the remark this answers is not in the document]
Comment (joemfb) [reply desk/app/contacts.hoon:404]: Oh man, I'm never gonna live this down ...

A self-poke should be enough to get out of the danger zone, right? The same event is fine, so long as all the apps have upgraded first?

**Comment by philipcmonk** [comment]
Oh right, that's correct

File: desk/app/contacts.hoon (hunk 1 above)
[the remark this answers is not in the document]
Comment (jamesacklin) [reply desk/app/contacts.hoon:404]: https://github.com/tloncorp/landscape-apps/pull/2189#issuecomment-1502705775

**Review by philipcmonk — APPROVED** [review]
Looks good

**Pull request merged**